gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.ts

```diff
@@ -48,11 +51,19 @@ export class PasswordPoliciesComponent implements OnInit {
       this.rows = policies;
     });
   }
+
   protected getTooltipText(id: string): string {
     const idpsNames = this.rows.find((pp) => pp.id === id).idpsNames;
     if (idpsNames === undefined || idpsNames.length === 0) {
       return null;
     }
     return 'Used in following Identity Providers: ' + idpsNames.join(', ');
   }
+
+  protected selectDefault(id: string) {
+    this.passwordPolicyService.update(this.domain.id, id, { defaultPolicy: true }).subscribe({
```

Comment:
**Suggestion**: I would expect to have a **setDefault** method which will call a dedicated endpoint on "password-policies" resoruce in a way that a single call will be responsible to reset the default flag and add the falg on the nex resource.